composer/src/lib.rs

```diff
@@ -939,6 +943,24 @@ impl ComposeTest {
         Ok(())
     }
 
+    /// disconnect container from the network
+    pub async fn disconnect(
+        &self,
+        container_name: &str,
+        network_name: &str,
```

Comment:
you already have the network name in self so you don't need to pass it as an argument

Reply:
Good point. I have replaced the use of _network_name_ for _self.network_id_.